scripts/bundle.config.js

```diff
@@ -67,7 +67,17 @@ const config = {
         // Compile ES2015 using babel
         test: /\.js$/,
         loader: 'babel-loader',
-        include: /src/
+        include: [/src/, /esm/],
```

Comment:
What is the `/esm/` doing here? Are we compiling the es5 from the esm dist?

Reply:
This is for the pre-built bundle. Keeping `forceAllTransforms` for backward compatibility.